### PR TITLE
Add bareditor

### PR DESCRIPTION
**Pull request by mrakitin**
Closes https://github.com/conda-forge/staged-recipes/issues/5369.

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/bareditor/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{% set name = "bareditor" %}
+{% set version = "0.17" %}
+{% set file_ext = "tar.gz" %}
```
Comment (hmaarrfk) [inline recipes/bareditor/meta.yaml]: I don't think you need the file ext and has variables. Just hard code them below. It makes things a little easier to read.

File: recipes/bareditor/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+test:
+  imports:
+    - bareditor
+    - bareditor.bareditor
+    - bareditor.imgdata
+
+about:
+  home: https://github.com/Prevalenter/bareditor
+  license: BSD-3-clause
+  license_family: BSD
```
Comment (hmaarrfk) [inline recipes/bareditor/meta.yaml]: You should open an issue upstream that they have an inconstent license in the setup.py

File: recipes/bareditor/meta.yaml (hunk 1 above)
Comment (mrakitin) [reply recipes/bareditor/meta.yaml]: Agreed, done.

File: recipes/bareditor/meta.yaml (hunk 2 above)
Comment (mrakitin) [reply recipes/bareditor/meta.yaml]: The GH issues are disabled for the project, so I went ahead and proposed a fix via a PR: https://github.com/Prevalenter/bareditor/pull/1.

**Comment by mrakitin** [comment]
Thanks for the review, @hmaarrfk! I addressed all your suggestions.

File: recipes/bareditor/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+{% set hash_type = "sha256" %}
+{% set hash_value = "6dbf897ff03a75182c23c3367dc110bf676857df9548e4bca6b0b452fdf0f432" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_value }}
```
Comment (hmaarrfk) [inline recipes/bareditor/meta.yaml]:
```suggestion
  sha256: 6dbf897ff03a75182c23c3367dc110bf676857df9548e4bca6b0b452fdf0f432
```

File: recipes/bareditor/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
+{% set name = "bareditor" %}
+{% set version = "0.17" %}
+{% set hash_type = "sha256" %}
```
Comment (hmaarrfk) [inline recipes/bareditor/meta.yaml]:
```suggestion
```

File: recipes/bareditor/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+{% set hash_value = "6dbf897ff03a75182c23c3367dc110bf676857df9548e4bca6b0b452fdf0f432" %}
```
Comment (hmaarrfk) [inline recipes/bareditor/meta.yaml]:
```suggestion
```

Comment (mrakitin) [reply recipes/bareditor/meta.yaml]: Can it be parametrized such as `{% set sha256 = "..." %}` and be used as `sha256: {{ sha256 }}`? I had a discussion about its convenience before in https://github.com/conda-forge/staged-recipes/pull/9739/files#r331470797, and was told it's OK to keep it as long as the name does not change. Having the name, version and sha256 in the beginning of the file makes maintenance more convenient.

Comment (hmaarrfk) [reply recipes/bareditor/meta.yaml]: if you think so, ok.

Comment (mrakitin) [reply recipes/bareditor/meta.yaml]: Sorry for the delay, lost a moment to address it on Sunday. 6b6db2e addresses it.

**Comment by mrakitin** [comment]
@hmaarrfk, I think this is ready to go. Could you please have another look?

**Comment by hmaarrfk** [comment]
I approved it. But somebody else will have to come and actually merge it in

**Comment by mrakitin** [comment]
@conda-forge/staged-recipes maintainers, could you please merge it?

**Comment by mrakitin** [comment]
Thank you!

**Pull request merged**